src/Components/src/GenericComponents/Icons.fs

```diff
@@ -0,0 +1,68 @@
+namespace Swate.Components
+
+open Feliz
+
+type Icons =
+
+    static member AddBuildingBlock =
+        React.fragment [
```

Comment:
Using `React.fragment` in this context might create issues with grouping. Try to replace it with `Html.span`.